docs/ispc.rst

```diff
@@ -2803,6 +2817,51 @@ them:
     const varying int x = { 1, 2, 3, 2 + 2 };
     const varying int y = x * 2;
 
+
+Attributes
+----------
+
+ISPC provides GNU style attributes syntax using ``__attribute__`` keyword.
+This section contains the list of currently supported attributes.
+
+noescape
+--------
+
+``__attribute__((noescape))`` can be placed on a function parameter of a
+uniform pointer type. It informs the compiler that the pointer cannot escape,
```

Comment:
The problem here is that varying pointer is not a pointer in LLVM IR. It is just a vector of `int`, while `nocapture` is applicable only to pointer types.

Reply:
So we should probably say something for varying pointer - that it's not supported or will have no effect.

Most likely it will cause questions from the users, as they don't understand implementation details.

Though I'm now sure how to best document it.

Reply:
I have added that it is not supported for varying pointers.